games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -82,5 +82,28 @@ index aaa3a5b..03723fb 100644
    InitExitMessageFunction(DisplayExitMessage);
    InitExitFunction(CloseAllAndExit);
 -- 
-1.8.3.4
+2.7.0
```

Comment:
If you want to have the smallest git diff, you can replace "2.7.0" by "1.8.2.4".